src/java.base/share/classes/jdk/internal/reflect/ClassFileAssembler.java

```diff
@@ -25,6 +25,8 @@
 
 package jdk.internal.reflect;
 
+import sun.nio.cs.UTF_8;
```

Comment:
I see about equal numbers of uses of `StandardCharsets.UTF_8` and `sun.nio.cs.UTF_8.INSTANCE` in java.base but would lean toward the one in `StandardCharsets` unless there's a bootstrap order dependency.